lib/twilioHelpers.js

```diff
@@ -0,0 +1,78 @@
+const twilio = require('twilio')
+
+const helpers = require('./helpers')
+
+// Set up Twilio
+const TWILIO_SID = helpers.getEnvVar('TWILIO_SID')
+const TWILIO_TOKEN = helpers.getEnvVar('TWILIO_TOKEN')
+const MessagingResponse = twilio.twiml.MessagingResponse
+
+function getTwilioClient() {
+  return twilio(TWILIO_SID, TWILIO_TOKEN)
+}
+
+function isValidTwilioRequest(request) {
+  return twilio.validateExpressRequest(request, TWILIO_TOKEN)
+}
+
+async function sendTwilioMessage(toPhoneNumber, fromPhoneNumber, message) {
+  const twilioClient = getTwilioClient()
+
+  try {
+    const response = await twilioClient.messages.create({
+      to: toPhoneNumber,
+      from: fromPhoneNumber,
+      body: message,
+    })
+
+    helpers.log(`Sent by Twilio: ${response.sid}`)
+
+    return response
+  } catch (err) {
+    helpers.logError(err.toString())
+  }
+}
+
+async function sendTwilioResponse(response, message) {
+  const twiml = new MessagingResponse()
+  twiml.message(message)
+  response.writeHead(200, { 'Content-Type': 'text/xml' })
+  response.end(twiml.toString())
+}
+
+async function buyAndConfigureTwilioPhoneNumber(areaCode, friendlyName) {
```

Comment:
Here is a generic version of the functions that you wrote to buy Twilio numbers for Sensors and Buttons. This can be used on both servers with their environment variables making the difference of which device it will buy the number for. What do you think @reesecritchlow ?

Reply:
I like it. Definitely reduces redundancy and keeps environment variables in one place.

Reply:
Hopefully pairing tool won't be used anymore.